pkg/vpath/helpers.go

```diff
@@ -0,0 +1,38 @@
+package vpath
+
+import (
+	"strings"
+)
+
+// pathWalker efficiently pops components off of a path and keeps track of the
+// amount of path consumed and remaining.
+type pathWalker struct {
+	path     string
```

Comment:
Any reason for not using `storj.Path` instead of `string`?

Reply:
Or perhaps, you need this is as a more efficient way to traverse through path components than using `storj.SplitPath`?

Reply:
`storj.Path` is a type alias to `string`, so they're really the same. The answer is I wrote this path walker a long time before trying to integrate it into the code base, and forgot to update it when I learned about `storj.Path`.